src/Materials/babylon.materialHelper.ts

```diff
@@ -103,6 +104,20 @@
                     }
                 }
 
+                if (light.lightmapMode != 0 ) {

```

Comment:
to ensure people know which value to use, I suggest also creating named values like here:
https://github.com/BabylonJS/Babylon.js/blob/master/src/babylon.engine.ts#L199